src/models/types.ts

```diff
@@ -304,8 +304,12 @@ type UsersBaseCollection = Collection<User> & {
 };
 
 export type UserCollection = UsersBaseCollection & {
+  createToken: (k: Knex, u: User, t: string) => Promise<SingleUseToken>
```

Comment:
Maybe all these token methods should be moved onto a dedicated `Tokens` collection

Reply:
To be completed in separate refactor PR